marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployDatabaseFieldCommand.java

```diff
@@ -141,6 +142,30 @@ protected void addExistingRangePathIndexes(Fragment newProps, Fragment existingP
         }
     }
 
+    protected void addExistingPathNamespaces(Fragment newProps, Fragment existingProps) {
+        Element newNamespaces = newProps.getInternalDoc().getRootElement().getChild("path-namespaces", MANAGE_NS);
+        if(newNamespaces == null) {
+            String esNamespace = "<path-namespaces><path-namespace><prefix>es</prefix><namespace-uri>http://marklogic.com/entity-services</namespace-uri>" +
+                    "</path-namespace></path-namespaces>";
+            newNamespaces = newProps.getInternalDoc().getRootElement().addContent(esNamespace);
+        }
+
+        List<String> newNamespacePrefixes = new ArrayList<>();
+        newProps.getElements("/m:database-properties/m:path-namespaces/m:path-namespace").forEach(namespace -> {
+            String prefix = namespace.getChildText("prefix", MANAGE_NS);
+            if (StringUtils.isNotBlank(prefix)) {
+                newNamespacePrefixes.add(prefix);
+            }
+        });
+
+        for (Element namespace : existingProps.getElements("/m:database-properties/m:path-namespaces/m:path-namespace")) {
+            String prefix = namespace.getChildText("prefix", MANAGE_NS);
+            if (StringUtils.isNotBlank(prefix) && !newNamespacePrefixes.contains(prefix)) {
```

Comment:
Same thing, don't need the isNotBlank check - if a path-namespace exists, it's guaranteed to have a prefix.